DateRangeBox: add code snippets and extended description #5043 (#5111)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDateRangeBox/1 Configuration/invalidEndDateMessage.md

---
---
##### shortDescription

> Specifies a message displayed if the typed end date is not a valid date. 

Reviewer: ```suggestion
Specifies a message for invalid end date input. 
```
